pages/installation/docker.rst

```diff
@@ -96,6 +98,9 @@ Example::
         # Password: admin
         - GRAYLOG_ROOT_PASSWORD_SHA2=8c6976e5b5410415bde908bd4dee15dfb167a9c873fc4bb8a81f6f2ab448a918
         - GRAYLOG_WEB_ENDPOINT_URI=http://127.0.0.1:9000/api
+      links:
+        - mongodb:mongo
+        - elasticsearch
```

Comment:
I tried your example on a rancher cluster and had two problems. Mongodb and es could not connect. Wish my changes everything came up nicely.